app/Models/Signup.php

```diff
@@ -22,7 +22,7 @@ class Signup extends Model
      * @var array
      */
     public static $indexes = [
-        'campaign_id', 'campaign_run_id',
+        'campaign_id', 'campaign_run_id', 'updated_at',
```

Comment:
@chloealee ahh I meant that it had to be made an index in a migration file, not added to the table. the example I linked is a migration creating a new index on an existing column